scripts/push_docs_to_branch.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+RASA_FOLDER=rasa
+TESTS_FOLDER=tests
+DATA_FOLDER=data
+TMP_DOCS_FOLDER=tmp-documentation
+TODAY=`date "+%Y%m%d"`
+# we build new versions only for majors
+PATTERN_FOR_NEW_VERSION="^refs/tags/[0-9]+\\.0\\.0$"
+MASTER_REF=refs/heads/master
+
+[[ ! $GITHUB_REF =~ $PATTERN_FOR_NEW_VERSION ]] \
+&& [[ $GITHUB_REF != $MASTER_REF ]] \
+&& echo "Not on master or major version, skipping." \
+&& exit 0
+
+NEW_VERSION=
+if [ "$GITHUB_REF" != $MASTER_REF ]
+then
+    NEW_VERSION=${GITHUB_REF/refs\/tags\//}
+fi
+
+# clone the $DOCS_BRANCH in a temp directory
+git clone --depth=1 --branch=$DOCS_BRANCH git@github.com:$GITHUB_REPOSITORY.git $TMP_DOCS_FOLDER
+
+echo "Updating the docs..."
+cp -R $DOCS_FOLDER/* $TMP_DOCS_FOLDER/docs
+# we need these because they are imported by docs files
+cp -R $RASA_FOLDER/* $TMP_DOCS_FOLDER/$RASA_FOLDER
+cp -R $DATA_FOLDER/* $TMP_DOCS_FOLDER/$DATA_FOLDER
+cp -R $TESTS_FOLDER/* $TMP_DOCS_FOLDER/$TESTS_FOLDER
+
+cd $TMP_DOCS_FOLDER
+
+if [ ! -z "$NEW_VERSION" ]
+then
+    echo "Generating docs for new version $NEW_VERSION..."
+    cd docs
+    yarn run new-version $NEW_VERSION
```

Comment:
I think there is another issue hidden here: it doesn't seem like docosaurus "scopes variables". This means that e.g. `{variables.release}` will get resolved to the same number for all versions.

Reply:
😱 wow this one, I didn't see it coming. I assumed that the `src` folder was moved inside each docs' version.

Reply:
I think I found a solution with c47348ed22b but it will require some testing

Reply:
aaaand, it works -> https://rasahq-docsv2-rasa.netlify.app/

Reply:
nice :+1: we prob need to do the same thing if we want to add e.g. the output of `rasa --help` to the docs

Reply:
yes I think so too